File: Java/Average_1546.java
import java.io.BufferedReader;
import java.io.InputStreamReader;
import java.util.StringTokenizer;

public class Average_1546 {
    private static int N;

    public static void main(String[] args) throws Exception {
        BufferedReader br = new BufferedReader(new InputStreamReader(System.in));
        N = Integer.parseInt(br.readLine());
        StringTokenizer st = new StringTokenizer(br.readLine()," ");

        float[] score = new float[N];
        float max = 0;
        for(int i=0; i<N; i++) {
            score[i] = Float.parseFloat(st.nextToken());
            if(score[i] > max) max = score[i];
        }

        float sum = 0;
        for(int i=0; i<N; i++) {
            score[i] = score[i] / max * 100;
            sum += score[i];
        }

        System.out.println(sum / N);
    }
}

File: C/Average_1546.cpp
#include <stdio.h>

int main() {
    int N;
    float max, sum;
    float score[1000];

    scanf("%d", &N);
    max = sum = 0;
    for(int i=0; i<N; i++) {
        scanf("%f", &score[i]);
        if(score[i] > max) max = score[i];
    }

    for(int i=0; i<N; i++) {
        score[i] = score[i] / max * 100;
        sum += score[i];
    }

    printf("%f", sum / N);
    return 0;
}